src/bot.py

```
@@ -104,39 +105,73 @@ def pull_data():
+            f"Cześć {user.mention_html()}, jestem botem wskaźnika jakości powietrza, pomogę Ci dowiedzieć się aktualnego AQI w {CITY}. "
+            f"Skorzystaj z /help, aby uzyskać więcej informacji!"
+        ),
     }
-    await update.message.reply_html(message[LANGUAGE],reply_markup=ForceReply(selective=True))
+    await update.message.reply_html(message[LANGUAGE])
+
 
 # Function to send answer to the /help command
 async def help(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
     """Send a message when the command /help is issued."""
     user = update.effective_user
```

Ruff (0.8.2)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)